src/main/java/Decide.java

```diff
@@ -448,20 +379,20 @@ boolean lic12() {
 
         return false;
     }
-    
+
     /**
-     * Returns true if there exists at least one set of three data points, 
+     * Returns true if there exists at least one set of three data points,
      * separated by exactly E_PTS and F_PTS consecutive
      * intervening points, respectively, that are the vertices of a triangle with area greater
      * than AREA1. In addition, there exist three data points (which can be the same or different
      * from the three data points just mentioned) separated by exactly E PTS and F PTS consecutive
      * intervening points, respectively, that are the vertices of a triangle with area less than
      * AREA2. Both parts must be true for the LIC to be true. Otherwise, return false.
-     * Also, return false when NUMPOINTS < 5. 
-     * 
+     * Also, return false when NUMPOINTS < 5.
+     * <p>
```

Comment:
What is this p tag here and in lots of other places?

Reply:
they where auto-generated by ctrl+alt+l, so just assumed that they belonged to the standard.